### PR TITLE
Allowing the tests fetch options to be passed an array

### PR DESCRIPTION
**Pull request by GlynnPhillips**
This will close https://github.com/nature/webpagetest-mapper/issues/15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -364,7 +364,7 @@ supporting the following properties:
   The WebPageTest connection speed.
   The default is `Native Connection`.
 * `tests`:
-  Path to the test definitions file.
+  Path to the test definitions file or an array.
```
Comment (GlynnPhillips) [inline README.md]: I am not sure on this wording, anyone got a better suggestion?

Comment (philbooth) [inline README.md]: I think `or an array of test definitions` would be better.

File: src/options.js [hunk 2]
```diff
@@ -183,9 +183,13 @@ function populateObject (object, defaultValues) {
 }
 
 function getTests (options) {
-    var tests = readJSON(options.tests, defaults.tests);
-
-    if (!options.results && !options.resultIds) {
+    var tests = options.tests;
+	
+	if(! Array.isArray(tests)) {
```
Comment (philbooth) [inline src/options.js]: Indentation in this project is four spaces in this project, not tabs. I realise that differs to some other Nature projects; are you using the DetectIndent vim plugin? It makes life easier.

Comment (GlynnPhillips) [inline src/options.js]: ahh I didn't catch this one. I thought I had updated everywhere I will fix this and start using DetectIndent

File: test/options.js [hunk 3]
```diff
@@ -600,6 +600,30 @@ suite('options:', function () {
                 });
             });
 
+            suite('normalise with tests array:', function () {
+                var normalised;
+
+                setup(function () {
+                    results.readFileSync[2] = '{"data":[],"options":{},"times":{"begin":"2015-01-04T14:55:08.577Z","end":"2015-01-04T14:55:08.578Z"}}';
+                    normalised = options.normalise({
+                        tests: ["a", "b", "c", "d"]
```
Comment (philbooth) [inline test/options.js]: Perfect! :grin:

**Comment by philbooth** [comment]
Looks good to me. :+1:

**Pull request merged**